net-irc/atheme-services/atheme-services-7.2.10_p2-r2.ebuild

```
@@ -27,7 +27,8 @@ RDEPEND="
 	perl? ( dev-lang/perl )
 	pcre? ( dev-libs/libpcre )
 	ssl? ( dev-libs/openssl:0= )"
-DEPEND="${RDEPEND}
+# 'dev-vcs/git' required as per bug #665802
+BDEPEND="${RDEPEND}
```

This is wrong, because everything in RDEPEND is also a DEPEND, not a BDEPEND.

```suggestion
DEPEND="${RDEPEND}"
BDEPEND="
```

---

So if my understanding of https://devmanual.gentoo.org/general-concepts/dependencies/ is correct then the original version is wrong because it doesn't specify what needs to be installed on the target system (`CHOST`), only what needs to be on the build system (`CBUILD`).  The runtime libraries should also be installed on the target system, and hence need to be included in `DEPEND`.

Anyways, I've gone ahead and made your suggested change.